TaskEstimation.md

```diff
@@ -0,0 +1,15 @@
+Task Estimation in Scrum
+
+Key Concepts:
+1.Sprint planning:
+This is a crucial part in a scrum meeting as this is where the team collaberates and determins whcich tasks can be done in the current sprint. The estimation is based off the resources and time that they have for the sprint. This practice would highlight communication and teamwork skills. According to wrike.com( https://www.wrike.com/scrum-guide/scrum-sprint-planning/) it states that the main importance of  sprint planning is "Sets clear defined goals, creates a communication platform and collaberation".
```

Comment:
Spelling error - "determins", "whcich", "collaberates"

remove extra space and change to "Sprint Planning"

reword the last line - it states that the main importance of  sprint planning is "Sets clear defined goals, creates a communication platform and collaberation".

Remove links and maybe add to end of your part?